Add example of parsing using a different starting rule (#155)
erikrose/parsimonious: README.rst

You can have forward references and even right recursion; it's all taken care
of by the grammar compiler. The first rule is taken to be the default start

> symbol, but you can start at any other rule as well.

Reviewer: ```suggestion
symbol, but the grammar will match any other rule as well.
```

Reviewer 2: To my ear, it sounds like that means the grammar will automatically try other rules if the starting one fails. But that's not the case.

Reviewer: ```suggestion
symbol, but you can use another rule as the start symbol by accessing it directly from the grammar (e.g. `grammar['text'].parse(...)`).
```

Might be a little verbose.

Reviewer 2: Sounds alright to me. I don't think there's a perfect place for it in the current, scant readme. It interrupts the flow of the easy case, but someone was wondering about it, so it's better to mention it than not. Just replace the single backtick with doubles (because that's ReStructuredText for "code"; single backticks are Markdown), and we're ready to fly.

        <Node called "bold_open" matching "((">
        <RegexNode called "text" matching "bold stuff">
        <Node called "bold_close" matching "))">

>     >>> print grammar['text'].parse('other stuff')

Reviewer: I think this should be removed. It clutters up the simple use case example. Once the documentation is expanded then we can better document use cases like this.

Reviewer: Please remove this.
